lldb/tools/lldb-dap/Handler/RequestHandler.h

```diff
@@ -113,6 +113,20 @@ class ExceptionInfoRequestHandler : public RequestHandler {
   void operator()(const llvm::json::Object &request) const override;
 };
 
+class GoToRequestHandler : public RequestHandler {
+public:
+  using RequestHandler::RequestHandler;
+  static llvm::StringLiteral getCommand() { return "goto"; }
```

Comment:
After rebasing on main... This function got renamed in 1028ea9e2641341436555347fb65d6f2eafcfbdd

```suggestion
  static llvm::StringLiteral GetCommand() { return "goto"; }
```